### ISSUE TITLE
Download file from Blob

### ISSUE THREAD

**Issue by Abhi2109** [Azure/azure-sdk-for-c#1659]
Hi,
I have a question about if we can download the file from blob using SDK APIs? Just like we have for upload.

**Comment by RickWinter** [comment]
@Abhi2109 Can you elaborate on the scenario.  Currently the C sdk does not provide storage apis.

**Comment by Abhi2109** [comment]
@RickWinter yes. So just like we have an API "IoTHubClientCore_LL_UploadToBlob" to upload the file in blob , can we have an API to download the same file from Blob in C.

**Comment by RickWinter** [comment]
@Abhi2109 Thank you for filling this issue and providing clarity.  I've added this to our backlog.

**Comment by Abhi2109** [comment]
Hi Rick,
Any idea by when this feature will be available?
DO i need to subscribe to any forum for getting the notification like "Ok this feature is available"

**Comment by RickWinter** [comment]
Blobs is available in beta
https://github.com/Azure/azure-sdk-for-c/blob/feature/storage-blobs/sdk/inc/azure/storage/az_storage_blobs.h
